examples/threathunter/threat_intelligence/results.py

```diff
@@ -0,0 +1,85 @@
+import enum
+import logging
+from datetime import datetime
+
+
+class IOC_v2():
+    """
+    Models an indicator of compromise detected during an analysis.
+
+    Every IOC belongs to an AnalysisResult.
+    """
+
+    def __init__(self, analysis, match_type, values, field, link):
+        self.id = analysis
+        self.match_type = match_type
+        self.values = values
+        self.field = field
+        self.link = link
+
+    class MatchType(str, enum.Enum):
+        """
+        Represents the valid matching strategies for an IOC.
+        """
+
+        Equality: str = "equality"
+        Regex: str = "regex"
+        Query: str = "query"
+
+
+    def as_dict(self):
+        return {
+            "id": str(self.id),
+            "match_type": self.match_type,
+            "values": list(self.values),
+            "field": self.field,
+            "link": self.link,
+        }
+
+
+class AnalysisResult():
```

Comment:
This appears to be the CB ThreatHunter report and the only normalization being done is on severity. Would it make sense to name this closer to `Carbon Black Report` or something?

Reply:
I kept the name AnalysisResult to keep the code generic enough for someone else to develop a connector and use results.py as a backbone for their TI